src/css/components/SelectCard.scss

```diff
@@ -0,0 +1,30 @@
+.Card {
+    border-radius: 5px;
+    width: 15%;
+    height: 25%;
+    text-align: center;
+    color: color-neutral(9);
+    border: 1px solid color-neutral(4);
+    font-size: $tu-base-fontSize;
+
+    &:hover {
+        box-shadow: 0 8px 16px 0 color-neutral(4);
+    }
+}
+
+.Card--container {
```

Comment:
I haven't been keeping up with CSS PRs lately, (and it's not noted anywhere easy to find!) but we should be using [SUIT CSS naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentname).

Example: 

`ComponentName` - primary class
`ComponentName-descendantElement` - descendant elements
`ComponentName--modifierName` - styles for modifiying main component
`Component.is-stateName` - state class

So in this case these should be descendents not modifiers: `Card-container`, `Card-icon` etc.

And in the `Card--selected` case, I think this should be a [state class](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentnameis-stateofcomponent) rather than a modifier.

So the class would be `.is-selected` and for specificity it would be styled as `Card.is-selected`.

Thank you and please reach out if you have any questions!

Reply:
@saralohr how long ago was the decision to use this made and was anyone who is currently working here who was a part of the decision still around?

Would like to know why this was chosen over BEM instead

Reply:
This is an alternate syntax for BEM, but still subscribes to the Block-Element-Modifier ideology, and all the CSS in TeamSnap UI _should_ follow it. I made the decision when we built V1 of TeamSnap UI.

Reply:
Now that we are relying more on React components the CSS class naming is less important, but still nice to be consistent if possible.